www/app.py

```diff
@@ -11,15 +12,20 @@
 except ImportError:
     print "Adafruit_GPIO lib unavailable"
 
-
+''' Default. Use with rovercode-web running at rovercode.com '''
 ROVERCODE_WEB_REG_URL = "https://rovercode.com/mission-control/rovers/"
+''' Uncomment this line to use with rovercode-web running at beta.rovercode.com '''
```

Comment:
Agreed that the "comment/uncomment" strategy is not good.
I looked into passing in a command line option, and it looks like it would be really tricky since uwsgi imports and runs the app.
How would you feel about reading it from a .env instead? I've seen other Flask apps that do that, and I could read it nicely using https://github.com/theskumar/python-dotenv.